test/extended/builds/s2i_root.go

```diff
@@ -37,7 +36,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds] s2i build with a root user imag
 		Before(oc)
 		defer After(oc)
 
-		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", image.LocationFor("registry.ci.openshift.org/ocp/4.7:test-build-roots2i"))
+		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", "quay.io/multi-arch/test-build-roots2i:ubi8")
```

Comment:
wait you switched things while I was working on my review ... this change is the same as https://github.com/openshift/origin/pull/26150/files :-)

that said, I hope based on my other explanation that if this passes, we then move forward with https://github.com/openshift/build-test-images/pull/7 this will change to 

image.LocationFor("registry.ci.openshift.org/ocp/4.8:test-build-roots2i"))

to pick up the (4.8 only changes) from https://github.com/openshift/build-test-images/pull/7